utils/create_fhir_store.sh

```diff
@@ -22,7 +22,34 @@ if [[ $# -ne 4 ]]; then
   exit 1
 fi
 
-# TODO create GCP Healthcare data-set too instead of assuming it exists.
+# Function that creates dataset and checks if the dataset already exist
+function create_dataset
+{
+  # Run command passed in
+  output=`$@ 2>&1`
+  
+  # Store the returned code
+  code=$?
+  
+  # If operation succeeded, output that dataset was created 
+  if [[ $code -eq 0 ]]; then
+      echo 'Created dataset'
+  else   
+    # If the dataset already exists, let user know
+    if [[ $output == *'already exists'* ]]; then
```

Comment:
nit (to be consistent with quotes)
```suggestion
    if [[ "${output}" == *"already exists"* ]]; then
```